dojo/importers/utils.py

```diff
@@ -168,3 +168,29 @@ def add_endpoints_to_unsaved_finding(finding, test, endpoints, **kwargs):
 def update_test_progress(test, **kwargs):
     test.percent_complete = 100
     test.save()
+
+
+def handle_vulnerability_references(finding):
+    # Synchronize the cve field with the unsaved_vulnerability_references
+    # We do this to be as flexible as possible to handle the fields until
+    # the cve field is not needed anymore and can be removed.
+    if finding.unsaved_vulnerability_references and finding.cve:
+        # Make sure the first entry of the list is the value of the cve field
+        finding.unsaved_vulnerability_references.insert(0, finding.cve)
+    elif finding.unsaved_vulnerability_references and not finding.cve:
+        # If the cve field is not set, use the first entry of the list to set it
+        finding.cve = finding.unsaved_vulnerability_references[0]
```

Comment:
you clearly fixed that the CVE should be always the first unsaved_vulnerability_references ?
if yes, what do we do if we don't have a CVE but another reference?

Reply:
If there is another reference (like GHSA or SNYK), it will be currently in the `cve` field, as well as being the first element of the vulnerability references, if the parser puts it there. There are parser who already did it before.

In the UI and the API this `cve` field is not used anymore, they are using the `Vulnerability_References`. It will be removed, after all parsers, the search and the hash_code algorithm are changed in subsequent PR's.